sources/api/netdog/src/networkd/config/network.rs

```diff
@@ -123,3 +128,45 @@ impl Display for DhcpBool {
         }
     }
 }
+impl NetworkConfig {
+    const FILE_EXT: &str = "network";
+
+    /// Write the config to the proper directory with the proper prefix and file extention
+    pub(crate) fn write_config_file(&self) -> Result<()> {
+        let cfg_path = self.config_path()?;
+
+        fs::write(&cfg_path, self.to_string()).context(error::NetworkDConfigWriteSnafu {
+            what: "network config",
+            path: cfg_path,
+        })
+    }
+    /// Build the proper prefixed path for the config file
+    fn config_path(&self) -> Result<PathBuf> {
+        let match_section = self
+            .r#match
```

Comment:
woah what is this syntax?

Reply:
The `r#match`?  Since `match` is a rust keyword you need to "escape" it in order to use it as an identifier.  "match" makes the most sense in this context, so I opted not to rename it.